shared/js/dataDriver.js

```diff
@@ -149,6 +152,13 @@ loop.DataDriver = function() {
             console.log(`Error handling EventSource.put: ${event.data}`, ex);
           }
           break;
+
+        // Handle an error, either unable to initially connect or
+        // unexpected disconnection
+        case "error":
```

Comment:
Repeating comment from before: 

I'm not super familiar with when EventSource will try to reconnect itself, but do we run into the potential problem of having multiple connections? E.g.,

1) the original connection gave an error but is also automatically trying to reconnect
2) multiple errors can happen so we end up starting multiple recursive calls

https://html.spec.whatwg.org/multipage/comms.html#reestablish-the-connection